src/lib/ui/AccessCodeTable/CodeDetails.tsx

```diff
@@ -0,0 +1,51 @@
+import dayjs from 'dayjs'
+import type { AccessCode } from 'seamapi'
+
+import { DotDivider } from 'lib/ui/layout/DotDivider.js'
+
+export function CodeDetails(props: { accessCode: AccessCode }) {
+  const { accessCode } = props
+
+  return (
+    <div className='seam-access-code-details'>
+      Unit 110
```

Comment:
Where do we need to load this from?

Reply:
That's the device name, no hooks implemented yet at the moment.